docs/guides/high-availability/deploy-aws-accelerator-stonith-lambda.adoc

```diff
@@ -0,0 +1,313 @@
+
+== Procedure
+. Enable Openshift user alert routing
++
+.Command:
+[source,bash]
+----
+kubectl apply -f - << EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    alertmanager:
+      enabled: true
+      enableAlertmanagerConfig: true
+EOF
+kubectl -n openshift-user-workload-monitoring rollout status --watch statefulset.apps/alertmanager-user-workload
+----
++
+. [[aws-secret]]Decide upon a username/password combination which will be used to authenticate the Lambda webhook and create an AWS Secret storing the password
++
+.Command:
+[source,bash]
+----
+aws secretsmanager create-secret \
+  --name webhook-password \ # <1>
+  --secret-string changeme \ # <2>
+  --region eu-west-1 # <3>
+----
+<1> The name of the secret
+<2> The password to be used for authentication
+<3> The AWS region that hosts the secret
++
+. Create the Role used to execute the Lambda.
++
+.Command:
+[source,bash]
+----
+<#noparse>
+FUNCTION_NAME= # <1>
+ROLE_ARN=$(aws iam create-role \
+  --role-name ${FUNCTION_NAME} \
+  --assume-role-policy-document \
+  '{
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Principal": {
+          "Service": "lambda.amazonaws.com"
+        },
+        "Action": "sts:AssumeRole"
+      }
+    ]
+  }' \
+  --query 'Role.Arn' \
+  --region eu-west-1 \#<2>
+  --output text
+)
+</#noparse>
+----
+<1> A name of your choice to associate with the Lambda and related resources
+<2> The AWS Region hosting your Kubernetes clusters
++
+. Create and attach the 'LambdaSecretManager' Policy so that the Lambda can access AWS Secrets
++
+.Command:
+[source,bash]
+----
+<#noparse>
+POLICY_ARN=$(aws iam create-policy \
+  --policy-name LambdaSecretManager \
+  --policy-document \
+  '{
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Effect": "Allow",
+              "Action": [
+                  "secretsmanager:GetSecretValue"
+              ],
+              "Resource": "*"
+          }
+      ]
+  }' \
+  --query 'Policy.Arn' \
+  --output text
+)
+aws iam attach-role-policy \
+  --role-name ${FUNCTION_NAME} \
+  --policy-arn ${POLICY_ARN}
+</#noparse>
+----
++
+. Attach the `ElasticLoadBalancingReadOnly` policy so that the Lambda can query the provisioned Network Load Balancers
++
+.Command:
+[source,bash]
+----
+<#noparse>
+aws iam attach-role-policy \
+  --role-name ${FUNCTION_NAME} \
+  --policy-arn arn:aws:iam::aws:policy/ElasticLoadBalancingReadOnly
+</#noparse>
+----
++
+. Attach the `GlobalAcceleratorFullAccess` policy so that the Lambda can update the Global Accelerator EndpointGroup
++
+.Command:
+[source,bash]
+----
+<#noparse>
+aws iam attach-role-policy \
+  --role-name ${FUNCTION_NAME} \
+  --policy-arn arn:aws:iam::aws:policy/GlobalAcceleratorFullAccess
+</#noparse>
+----
++
+. Create a Lambda ZIP file containing the required STONITH logic
```

Comment:
`STONITH` reference could be better represented as pointed out by @ahus1 earlier.